UnitTests/GitUI.Tests/UserControls/FilterInfoTests.cs

```diff
@@ -741,5 +741,101 @@ public void FilterInfo_GetRevisionFilter(string author, string committer, string
             filterInfo.GetSummary().Should().Be(expectedSummary);
             filterInfo.GetRevisionFilter().ToString().Should().Be(expectedArgs);
         }
+
+        [Test]
+        public void FilterInfo_Apply_ByCommitMessage()
+        {
+            FilterInfo filterInfo = new();
+            var filterLaunched = filterInfo.Apply(new RevisionFilter("message", true, false, false, false));
```

Comment:
Please add arg names to help with readability

Reply:
Sorry, resharper helps a lot with that...